### PR TITLE
feat: allow to select from registry list if name is undefined

### LINKED ISSUE #187: [Improvement] nrm use supports select
# QUESTION

Sometimes, I don't remember the list of npm registries. While using `nrm use` command,  I don't enter registry name alias, it will show error.  How about changing this behavior? If you don't enter registry npm alias, the terminal can provide a registry list for user to select. I think it can improve development experience.

### PR DESCRIPTION
**Pull request by chouchouji**
# Issue

close #187 

# Result

https://github.com/user-attachments/assets/2c55425a-690b-4377-bca7-140f5efdb882

### CODE REVIEW

**Comment by chouchouji** [comment]
Thanks @iosh. The test ci worked well.

**Comment by iosh** [comment]
OK, I will add test to this PR (in this weekend)

**Comment by chouchouji** [comment]
> OK, I will add test to this PR (in this weekend)

Thanks again. 👍🏻

**Comment by chouchouji** [comment]
I had merged the pr into this branch. Thanks! But the test ci failed. 

image

@iosh I think this module can be removed. If you think so, please fix it by this branch.

**Comment by chouchouji** [comment]
I fix it.  emmmmm

image

File: tests/cli.test.js [hunk 1]
```diff
@@ -58,6 +58,26 @@ it('nrm use <registry>', async () => {
...
+  const { stdout } = spawn('nrm', ['use'], { shell: isWin });
+
+  const message = await new Promise((resolve) => {
+    stdout.on('data', (data) => {
+      resolve(stripAnsi(data.toString()).trim());
+    });
+  });
+
+  expect(message).toBe(`? Please select the registry you want to use (Use arrow keys)
+${isWin ? '>' : '❯'} npm
```
Comment (chouchouji) [inline tests/cli.test.js:71]: @iosh It is used to adapt to mac and windows. Otherwise, test ci will fail....

**Pull request merged**